pootle/apps/import_export/views.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2009-2015 Zuza Software Foundation
+#
+# This file is part of Pootle.
+#
+# Pootle is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# Pootle is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# Pootle; if not, see <http://www.gnu.org/licenses/>.
+
+import os
+from io import BytesIO
+from zipfile import ZipFile, is_zipfile
+from django.core.servers.basehttp import FileWrapper
+from django.http import Http404, HttpResponse
+from django.utils.translation import ugettext as _
+from translate.storage import po
+from pootle_store.models import Store
+from .forms import UploadForm
```

Comment:
Please review [the styleguide on imports](http://docs.translatehouse.org/projects/pootle/en/latest/developers/styleguide.html#imports).